src/ui/pages/GenericInfoPageWhite.tsx

```diff
@@ -53,11 +50,6 @@ export const GenericInfoPageWhite: React.FC<Props> = ({
   const { canGoBack, goBack } = useGoBack(...(props.goBackParams ?? homeNavConfig))
   const animationRef = useRef<LottieView>(null)
   const grid = useGrid()
-  const isSmallHeight = useMediaQuery({ maxHeight: SMALL_HEIGHT })
```

Comment:
I deleted this part since:
- isSmallScreen == true -> lottieStyle is { height: '100%' } from lottieStyle
- isSmallScreen == false -> lottieStyle is { height: '100%' } from the LottieView styled component